src/components/DefinitionEntry.js

```diff
@@ -71,7 +71,6 @@ export default class DefinitionEntry extends React.Component {
   renderHeadline(definition, curation) {
     const { namespace, name, revision } = definition.coordinates
     const namespaceText = namespace ? namespace + '/' : ''
-    const scores = Definition.computeScores(definition)
```

Comment:
perhaps just change this to `const scores = get(definition, 'scores')` then all the other code changes can be reverted and the duplicate code removed. 

The same suggestion applies to `HeaderSection` and `SystemManagedList`